manifests/config.pp

```diff
@@ -40,15 +40,7 @@
     }
   }
 
-  $_config.each |$setting, $value| {
-    ini_setting { "${config_file} ${setting} ${value}":
-      ensure  => present,
-      path    => $config_file,
-      section => '',
-      setting => $setting,
-      value   => $value,
-      require => File['/etc/letsencrypt'],
-    }
-  }
+  $_config_joined = join_keys_to_values($_config, '=')
+  letsencrypt::config::ini { $_config_joined: }
```

Comment:
We could probably use [`create_ini_settings`](https://github.com/puppetlabs/puppetlabs-inifile/blob/master/lib/puppet/parser/functions/create_ini_settings.rb) for this.